Jenkinsfile

```diff
@@ -0,0 +1,92 @@
+library 'magic-butler-catalogue'
+def PROJECT_NAME = 'logdna-logspout'
+def TRIGGER_PATTERN = ".*@logdnabot.*"
+def DEFAULT_BRANCH = 'master'
+def CURRENT_BRANCH = [env.CHANGE_BRANCH, env.BRANCH_NAME]?.find{branch -> branch != null}
+
+pipeline {
+  agent {
+    node {
+      label 'ec2-fleet'
+      customWorkspace "${PROJECT_NAME}-${BUILD_NUMBER}"
+    }
+  }
+
+  options {
+    timestamps()
+    ansiColor 'xterm'
+  }
+
+  triggers {
+    issueCommentTrigger(TRIGGER_PATTERN)
+  }
+
+  stages {
+    stage('Validate PR Source') {
+      when {
+        expression { env.CHANGE_FORK }
+        not {
+          triggeredBy 'issueCommentCause'
+        }
+      }
+      steps {
+        error("A maintainer needs to approve this PR for CI by commenting")
+      }
+    }
+    stage('Lint') {
+      steps {
+        sh 'make lint'
+      }
+    }
+
+    stage('Release') {
+      when {
+        branch "${DEFAULT_BRANCH}"
+        not {
+          changelog '\\[skip ci\\]'
+        }
+      }
+
+      environment {
+        GITHUB_TOKEN = credentials('github-api-token')
+        JENKINS_URL = "${JENKINS_URL}"
+        CHANGE_ID = ''
+        USERNAME = 'logdna'
+      }
+
+      stages {
+        stage('Create Release') {
+          tools {
+            nodejs 'NodeJS 14'
+          }
+
+          steps {
+            versioner(
```

Comment:
curious why you ended up using semantic-release

based on our slack conversation, i thought you were going to use `svu`